Monika-After-Story/MonikaModDev
# Adding twin ribbon spritepacks

#4127 

# TODO
- [x] select dlg

# Testing
* verify that the twin ribbons can only be worn with a twintail hair style (twintails/usagi)
* verify that the twin ribbons look good
